AmbiguityVectorizer.py

```diff
@@ -1,6 +1,8 @@
 from nltk import pos_tag_sents
```

Comment:
1.please add some word explaining what does this class do 
2.add function documentation - we should start in some code standards  i think

Reply:
added documentation for all